components/ai_chat/resources/page/components/input_box/index.tsx

```diff
@@ -77,9 +82,20 @@ function InputBox(props: InputBoxProps) {
   }
 
   const maybeAutofocus = (node: HTMLTextAreaElement | null) => {
-    if (node && props.context.selectedActionType) {
+    if (!node) {
+      return
+    }
+
+    if (props.context.selectedActionType) {
       node.focus()
     }
+    if (props.context.isMobile && props.context.hasAcceptedAgreement &&
```

Comment:
I'm thinking this should be done in a higher level because input is shared with ai rewriter dialog - it doesn't need to be in a conversation with history and doesn't need to have conversation awareness. We can also remove the `props.context.isMobile`

Reply:
I moved it to the higher level